### PR TITLE
Enhance zypper install error messages.

### PR DESCRIPTION
**Pull request by ycedres**
### What does this PR do?
Adds stdout to the error message if zypper install returns with an error code and there is no information on standard error.

### What issues does this PR fix or reference?
None. 

### Previous Behavior
If zypper install returns with an error code and there is no output on standard error, tells the user to check zypper logs.

### New Behavior
If zypper install returns with an error code and there is no output on standard error, uses standard output to give more information to the user.

### Merge requirements satisfied?

- [X] Docs
- [X] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [X] Tests written/updated

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: changelog/62346.changed [hunk 1]
```diff
@@ -0,0 +1 @@
+Enhance zypper install error messages.
```
Comment (meaksh) [inline changelog/62346.changed:1]: I think these changes are affecting not only "install" method, right? Maybe a better changelog message would be:

```suggestion
Enhance capture of error messages for Zypper calls in zypperpkg module.
```

Comment (ycedres) [reply changelog/62346.changed:1]: Absolutely right!

**Review by meaksh — DISMISSED** [review]

**Comment by ycedres** [comment]
Some checks unrelated to the PR keep failing so I will wait some days to try to run them again.

**Review by meaksh — DISMISSED** [review]

**Comment by ycedres** [comment]
@krionbsd some checks keep failing on this PR that are apparently unrelated to the changes. I've tried forcing a rerun several times and different checks failed each time. Do you have any idea about what could be happening?

**Review by Ch3LL — APPROVED** [review]

**Pull request merged**